src/App.tsx

```diff
@@ -15,71 +15,86 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  clearButton = () => {
+    this.setState({ selectedGood: '' });
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  addButton(
+    _event: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+    name:string,
```

Comment:
You don't use this event, you can invoke the function without it
```suggestion
  addButton (name:string,
```